src/routes/write-exercise/components/AnswerReview.tsx

```diff
@@ -55,6 +55,10 @@ const Feedback = ({ documentWithResult, submission, needsToBeRepeated }: Feedbac
     message = `${getLabels().exercises.write.feedback.almostCorrect1} „${submission}“ ${
       getLabels().exercises.write.feedback.almostCorrect2
     }`
+  } else if (!submission) {
```

Comment:
All these changes from your other branch LUN-417 appear here, which makes it difficult to review. Please adjust the base branch of this feature branch, see here:
https://stackoverflow.com/questions/10853935/change-branch-base